modelservers/resnet50/README.md

```diff
@@ -0,0 +1,99 @@
+# Create a pretrained ResNet50 model server and run a load test
+
+#### 1. Create the model
+
+##### Using a Spell Run
```

Comment:
I think we should just show a single way to keep this as straightforward as possible. I'd vote upload.